config-solaris.md

```diff
@@ -90,7 +90,7 @@ Mapped to `allowed-address` in the [zonecfg(1M)][zonecfg.1m_2] man page.
 * **`configureAllowedAddress`** *(string, OPTIONAL)* If configureAllowedAddress is set to true, the addresses specified by allowedAddress are automatically configured on the interface each time the container starts.
 When it is set to false, the allowedAddress will not be configured on container start.
 Mapped to `configure-allowed-address` in the [zonecfg(1M)][zonecfg.1m_2] man page.
-* **`defrouter`** *(string, OPTIONAL)* The value for the OPTIONAL default router.
+* **`defaultRouter`** *(string, OPTIONAL)* The value for the OPTIONAL default router.
```

Comment:
The link in the solaris markdown was wrong, but I'm updating it to http://docs.oracle.com/cd/E19253-01/816-5168/priv-str-to-set-3c/index.html as that seems the correct link.

However, as from that link the documentation seems to use `Priv`, I'm leaving it as-is.

As for `def` or `default`, `def` seems a very unusual abbreviation. Using `default` is also consistent with `defaultAction`. There seems no point that I can see for abbreviating it.